### PR TITLE
SEO-178943-React-Page-With-Redirect2

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on links with redirects.

Kind Regards,
Beryl.

### CODE REVIEW

File: ej2-react/multi-select/checkbox.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ Using [`enableSelectionOrder`](https://ej2.syncfusion.com/react/documentation/ap
 
 ## See Also
 
-* [How to bind the data](./data-binding/)
-* [How to filter the bound data](./filtering/)
-* [How to add custom value to the MultiSelect](./custom-value/)
+* [How to bind the data](./data-binding)
+* [How to filter the bound data](./filtering)
+* [How to add custom value to the MultiSelect](./custom-value)
 * [How to render grouping with checkbox](./grouping/#grouping-with-checkbox).
```
Comment (Yvone-Atieno) [inline ej2-react/multi-select/checkbox.md:142]: Check the above link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-react/multi-select/checkbox.md (hunk 1 above)
Comment (berylouma) [reply ej2-react/multi-select/checkbox.md]: Updated

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request merged**